src/bench/bench.cpp

```diff
@@ -57,6 +57,10 @@ void benchmark::BenchRunner::RunAll(const Args& args)
     std::regex reFilter(args.regex_filter);
     std::smatch baseMatch;
 
+    if (args.sanity_check) {
+        std::cout << "Running with --sanity check option, benchmark results will be useless." << std::endl;
```

Comment:
```suggestion
        std::cout << "Running with -sanity-check option, benchmark results will be useless." << std::endl;
```

Also s/--sanity-check/-sanity-check/ in the commit message?

Reply:
I'm not sure as there isn't a single-letter option like -s here,  but per https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html: "Long options consist of -- followed by a name made of alphanumeric characters and dashes. Option names are typically one to three words long, with hyphens to separate words. Users can abbreviate the option names as long as the abbreviations are unique."

Reply:
Looks like "--sanity check" in this line should be "sanity check" or "--sanity-check".

Reply:
Just a missing hyphen s/--sanity check/--sanity-check/